rules/docker-fpm-quagga.mk

```diff
@@ -11,3 +11,4 @@ $(DOCKER_FPM_QUAGGA)_RUN_OPT += --privileged -t
 $(DOCKER_FPM_QUAGGA)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
 
 $(DOCKER_FPM_QUAGGA)_BASE_IMAGE_FILES += vtysh:/usr/bin/vtysh
+$(DOCKER_FPM_QUAGGA)_BASE_IMAGE_FILES += $(SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT)
```

Comment:
Check builds are failing because `SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT` should be added to `$(DOCKER_FPM_QUAGGA)_FILES`, not `$(DOCKER_FPM_QUAGGA)_BASE_IMAGE_FILES`

Reply:
Fixed.